test/hash_is_avalanching_test.cpp

```diff
@@ -41,5 +58,10 @@ int main()
     BOOST_TEST_TRAIT_FALSE(( hash_is_avalanching< boost::hash<std::basic_string<my_char> > > ));
 #endif
 
+    BOOST_TEST_TRAIT_FALSE(( hash_is_avalanching< std::hash<std::string > > ));
```

Comment:
Why are you testing this?

Reply:
It's an example of a struct without a nested `is_avalanching`.